packages/@okta/vuepress-site/docs/concepts/direct-authentication/index.md

```diff
@@ -0,0 +1,102 @@
+
+In direct authentication, the client app communicates directly with the authentication service to validate user credentials. This method is useful in scenarios where usability constraints hinder the use of browser-based flows, such as with mobile apps. In addition, it's beneficial in situations where speed and simplicity are important, such as with single-page applications (SPAs). By eliminating the need for redirects to external identity providers, direct authentication reduces latency and enhances the overall user experience.
+
+### Provides more control
+
+Direct auth gives you more control over the authentication process by handling authentication internally in your app. You can tailor security measures to your needs without relying on third-party providers. Keep in mind that this also places a greater responsibility on you to ensure the secure handling of user credentials. Use direct authentication only in situations where there's a high degree of trust between the user and your app.
+
+## Key Characteristics
+
+Direct authentication stands apart from traditional methods due to several defining characteristics that streamline the user experience and simplify implementation in specific scenarios:
+
+* **Client-initiated credential exchange**: The client app directly sends user credentials (for example, username/password, API key, token) to the service's authentication endpoint.
+* **API-centric**: Authentication is typically performed using RESTful APIs.
+* **Minimal redirection**: The need for browser redirects to IdPs is reduced or eliminated. This keeps the user within the app's context.
+* **Direct response**: The authentication service directly responds to the client with an authentication token (for example, JWT or OAuth token) upon successful validation.
+* **Enhanced user experience**: The sign-in experience is faster, more fluid, and less disruptive.
+
+## How it Works
+
+At a high level, the direct auth flow has the following steps:
+
+1. Your client app prompts the user for their credentials (username and password) and then with an authenticator, if required.
+2. The user enters their credentials in the app (for example, mobile app, web form), and then addresses an authenticator prompt in the app, if required.
+3. Your app builds the API request that contains these credentials and sends it directly to the Okta  `/token` endpoint.
+
+    **Note**: This is a high-level example. There are other requests and responses that can occur between your app, the user, and Okta. This depends on the types of authenticators that you configure. See [Configure Direct Authentication](/docs/guides/configure-direct-auth-grants/aotp/main/) for example authenticator flows that Okta supports.
+
+4. Okta receives the request and validates the credentials. If validation is successful, Okta generates and returns an access token to your app.
```

Comment:
```suggestion
4. Okta receives the request and validates the credentials. If validation is successful and the application's assurance requirements are met, Okta generates and returns an access token to your app. Otherwise, Okta responds saying more assurance is needed.
```